lib/web_ui/lib/src/engine/text/measurement.dart

```diff
@@ -737,10 +731,13 @@ class LinesCalculator {
   /// The lines that have been consumed so far.
   List<EngineLineMetrics> lines = <EngineLineMetrics>[];
 
-  int _lineStart = 0;
-  int _chunkStart = 0;
+  int get _lineStart => _lastTakenBreak?.index ?? 0;
+  int get _chunkStart => _lastBreak?.index ?? 0;
   bool _reachedMaxLines = false;
 
+  LineBreakResult? _lastBreak;
+  LineBreakResult? _lastTakenBreak;
```

Comment:
Instead of making these nullable is it possible to default them to a `const` value with index of `-1` or `0`? It would remove null checks and null assertions.

Reply:
Ah I see your point now about making `LineBreakResult` a const. Makes sense.